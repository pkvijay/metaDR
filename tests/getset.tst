get BADKEY
set BADKEY 123
create /test 123
get /test
set /test foo
get /test
get /missing_key
set /missing_key bar
