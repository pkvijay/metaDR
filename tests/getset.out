MALFORMED KEY
MALFORMED KEY
CREATED
123
foo
KEY NOT FOUND
KEY NOT FOUND
